test/passing/tests/attributes.ml

```diff
@@ -11,7 +11,9 @@ val x : ?x:unit (** not dropped *) -> unit
 type t =
   { a: int
   ; b: int [@default 1] [@drop_if]
-  ; c: int [@default 1] [@drop_if]
+  ; c: int
+        [@default 1]
+        [@drop_if]
         (** docstring that is long enough to break *) }
```

Comment:
I think docstrings shouldn't interfere with attributes. This box shouldn't be removed.

Reply:
reverted